src/ensembl/src/content/app/browser/track-panel/track-panel-modal/modal-views/TrackPanelBookmarks.test.tsx

```diff
@@ -25,12 +26,25 @@ import { TrackPanelBookmarks } from './TrackPanelBookmarks';
 
 import { PreviouslyViewedObject } from '../../trackPanelState';
 
-jest.mock('react-router-dom', () => ({
-  Link: (props: any) => (
-    <div {...props} className={'link'}>
-      {props.children}
-    </div>
-  )
+jest.mock('react-redux', () => ({
+  useSelector: jest.fn().mockImplementation((selector) => selector()),
+  useDispatch: jest.fn()
+}));
+
+jest.mock('src/shared/state/ens-object/ensObjectSelectors', () => ({
+  getExampleEnsObjects: jest.fn()
+}));
```

Comment:
Here and below, please refer to the point number 2 mentioned by Andrey here:
https://github.com/Ensembl/ensembl-client/pull/435#discussion_r567514153

Reply:
✅